Onboarding/Connector/Scripts/Documentation/BashScriptTsgAndFaq.md

```diff
@@ -0,0 +1,90 @@
+# For the bash script
+
+## FAQ
+
+### How do I run the script?
+
+Use the below commands
+
+For single subscription
+
+```bash
+chmod +x create-impact-reporting-connector.sh
+./create-impact-reporting-connector.sh --subscription-id <subscription_id>
+```
+
+For new line separated subscription list in a file
+
+```bash
+chmod +x create-impact-reporting-connector.sh
+./create-impact-reporting-connector.sh --file-path <file_path>
+```
+
+### Can I run this script for multiple subscriptions at once?
+
+Yes, use the `--file-path` option with a file containing newline-separated subscription IDs.
+
+### What permissions are needed to run this script?
+
+You need to have **Contributor** permission to log in to Azure, register resource providers, and create connectors in the Azure subscriptions. You also need to have either of **'Owner'**, **'User Access Administrator'** and **'Privileged Role Administrator'** permissions to create and assign custom roles.
+
+### How do I enable debug mode?
+
+Uncomment the `set -x` at the beginning of the script to enable debug mode.
+
+### What should I do if I encounter a permission error?
+
+Verify your Azure role and permissions. You may need the help of your Azure administrator to grant you the necessary permissions or roles as defined the permissions section.
+
+### How can I verify if the connector is successfully created?
+
+Run the below command
+
+```bash
+az rest --method get --url "https://management.azure.com/subscriptions/<subscription-id>/providers/Microsoft.Impact/connectors?api-version=2024-05-01-preview"
+```
+
+You should see a resource with the name: **AzureMonitorConnector**
```

Comment:
We should also give the option of checking via ARG

Reply:
Done